src/main/java/com/microsoft/cse/helium/app/config/CosmosDbConfig.java

```diff
@@ -18,7 +18,9 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
+
```

Comment:
Are empty lines part of a standard?  If not, let's remove empty lines.

Reply:
Sure!